corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -38,30 +39,71 @@
     get_single_worksheet,
 )
 
+from corehq.util.files import read_workbook_content_as_file
 
-def validate_bulk_app_translation_upload(app, workbook, email, lang_to_compare):
+from CommcareTranslationChecker import validate_workbook
+from CommcareTranslationChecker.exceptions import FatalError
+
+
+def run_translation_checker(file_obj):
+    translation_checker_messages = []
+    result_wb = None
+    try:
+        result_wb = validate_workbook(file_obj, translation_checker_messages)
```

Comment:
as a follow up improvement or if you can do it now, can we update `validate_workbook` to not need any empty list to be passed and just return the messages itself, so this would become
```
    result_wb, translation_checker_messages = validate_workbook(file_obj)
```